docker-for-windows/index.md

```diff
@@ -164,57 +166,55 @@ running something more complex, such as an OS and a webserver.
     ```
 
-8.  Stop the running nginx container by the name we assigned it, `webserver`:
+8. Stop the running nginx container by the name we assigned it, `webserver`:
 
     ```shell
     >  docker container stop webserver
     ```
 
-9.  Remove all three containers by their names -- the latter two names will differ for you:
+9. Remove all three containers by their names -- the latter two names will differ for you:
 
     ```shell
     > docker container rm webserver laughing_kowalevski relaxed_sammet
     ```
 
 ## Docker Settings dialog
 
-The **Docker Desktop for Windows menu** is a popup by which you can configure your
-Docker settings -- installation, updates, version channels, Docker Hub login,
+The **Docker Desktop** menu allows you to configure your Docker settings such as installation, updates, version channels, Docker Hub login,
 and more.
 
 This section explains the configuration options accessible from the **Settings** dialog.
 
-1.  Open the Docker Desktop for Windows menu by right-clicking the Docker icon in the Notifications area (or System tray):
+1. Open the Docker Desktop menu by clicking the Docker icon in the Notifications area (or System tray):
 
     ![Showing hidden apps in the taskbar](images/whale-icon-systray-hidden.png){:width="250px"}
 
-2.  Select **Settings...** to open the Settings dialog:
+2. Select **Settings** to open the Settings dialog:
 
-    ![Docker Desktop for Windows popup menu](images/docker-menu-settings.png){:width="400px"}
+    ![Docker Desktop popup menu](images/docker-menu-settings.png){:width="300px"}
 
 ### General
 
 On the **General** tab of the Settings dialog, you can configure when to start and update Docker.
 
 ![Settings](images/settings-general.png){:width="600px"}
 
-* **Start Docker when you log in** - Automatically start the Docker Desktop for Windows
-application upon Windows system login.
+* **Start Docker when you log in** - Automatically start Docker Desktop upon Windows system login.
 
-* **Automatically check for updates** - By default, Docker Desktop for Windows
-automatically checks for updates and notifies you when an update is available.
+* **Automatically check for updates** - By default, Docker Desktop automatically checks for updates and notifies you when an update is available.
 Click **OK** to accept and install updates (or cancel to keep the current
 version). You can manually update by choosing **Check for Updates** from the
 main Docker menu.
 
-* **Send usage statistics** - By default, Docker Desktop for Windows sends diagnostics,
+* **Send usage statistics** - By default, Docker Desktop sends diagnostics,
 crash reports, and usage data. This information helps Docker improve and
-troubleshoot the application. Uncheck to opt out. Docker may also  sometimes
-prompt you for more information.
+troubleshoot the application. Clear the check box to opt out. Docker may also  sometimes prompt you for more information.
```

Comment:
Looks like there is an extra space between "also" and "sometimes."

Reply:
Good spot. Removed the redundant space. Updated the text as suggested.